src/tools/qt-gui/README.md

```diff
@@ -65,3 +76,29 @@ The following command will launch the Qt-GUI:
 ```sh
 kdb qt-gui
 ```
+
+After that a window will open that looks something like that:
+
+![empty GUI](src/tools/qt-gui/images/Qt-GUI-1.png)
```

Comment:
The links to the images are not working (see overall review).

Reply:
Thank you for pointing out, I think they are fixed now!